tracks/julia/exercises/difference-of-squares/mentoring.md

```diff
@@ -4,6 +4,24 @@
 
 ![](https://wikimedia.org/api/rest_v1/media/math/render/svg/5852a9aaf138078bfb5a60e1e6cad558b57a9c3e)
 
+- Analytical solutions for sum of the first N square numbers: [square pyramidal number](https://en.wikipedia.org/wiki/Square_pyramidal_number)
+
 ### Talking points
 
+- You might be interested to know that `sum(1:n)` already uses Gauss' formula. You can see this with `@edit sum(1:n)`
 - For integer division, students should use `÷` and not `/` (see [#138](https://github.com/exercism/julia/issues/138))
+
+### Interesting solutions
+
+n111b111's solution:
```

Comment:
Perhaps adding a link to their solution on exercism.io makes it easier to link this to students? I'm not sure if they can access it though

Reply:
I prefer to give it inline to avoid having to open a other page to see.

Also, users can change their solutions to something else whenever they like.